<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GpsInfoWidget</class>
 <widget class="QWidget" name="GpsInfoWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>218</width>
    <height>186</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>218</width>
    <height>170</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
	background-color: rgba(0,0,0,0.8);
	color: #FFF;
	font-weight: bold;
}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="rightMargin">
    <number>18</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Positionnement GPS</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="xLabel">
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>X :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="yLabel">
          <property name="text">
           <string>Y :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="altitudeLabel">
          <property name="text">
           <string>Altitude :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="satellitesLabel">
          <property name="text">
           <string>Satellites :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="dgpsLabel">
          <property name="text">
           <string>DGPS :</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
